packages/headless/package.json

```diff
@@ -48,12 +48,14 @@
   "dependencies": {
     "@coveo/bueno": "0.43.7",
     "@reduxjs/toolkit": "1.8.5",
+    "@types/event-source-polyfill": "^1.0.1",
     "@types/pino": "6.3.12",
     "@types/redux-mock-store": "1.0.3",
     "abab": "2.0.6",
-    "coveo.analytics": "2.28.5",
+    "coveo.analytics": "2.28.7",
     "cross-fetch": "3.1.5",
     "dayjs": "1.11.5",
+    "event-source-polyfill": "^1.0.31",
```

Comment:
How much thoughts/analysis have we put into checking how the polyfill works, it's size, who's the developer, when and why it's needed, etc.

Any dependency we add here has to be very solid.

It seems that https://caniuse.com/eventsource is supported by all browsers we care about ?
And so we'd only need it for Node JS (or do we ?)